### PR TITLE
spring boot 3.0.0

### PR DESCRIPTION
**Pull request by MarcinNowak-codes**
Thank you for submitting a pull request to the WrongSecrets app!

### CODE REVIEW

File: src/main/java/org/owasp/wrongsecrets/MvcConfiguration.java [hunk 1]
```diff
@@ -10,9 +10,12 @@
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
```
Comment (nhumblot) [inline src/main/java/org/owasp/wrongsecrets/MvcConfiguration.java:13]: :wave: 

This import is not used, could you please remove it? :slightly_smiling_face:

File: src/test/java/org/owasp/wrongsecrets/HerokuWebSecurityConfigTest.java [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.client.ResourceAccessException;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
 
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+// @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```
Comment (nhumblot) [inline src/test/java/org/owasp/wrongsecrets/HerokuWebSecurityConfigTest.java:14]: :wave: 

This code has been commented, wouldn't it be better to remove it? :thinking:

**Review by nhumblot — COMMENTED** [review]
Thank you for your changes! :hugs: 

I just found a couple of minor things on my side.

Running the application on my workstation looks good, I am not facing any issue, everything looks to be fine. Congrats! :clap:

File: src/main/java/org/owasp/wrongsecrets/MvcConfiguration.java (hunk 1 above)
Comment (MarcinNowak-codes) [reply src/main/java/org/owasp/wrongsecrets/MvcConfiguration.java]: Removed :-)

File: src/test/java/org/owasp/wrongsecrets/HerokuWebSecurityConfigTest.java (hunk 2 above)
Comment (MarcinNowak-codes) [reply src/test/java/org/owasp/wrongsecrets/HerokuWebSecurityConfigTest.java]: I opened ticket https://github.com/spring-projects/spring-boot/issues/33451 for RestTemplate issue. This is example how to reproduce `Server redirected too many times`.

**Pull request merged**